client/lcd/root.go

```diff
@@ -149,9 +144,9 @@ func createHandler(cdc *codec.Codec) *mux.Router {
 	rpc.RegisterRoutes(cliCtx, r)
 	tx.RegisterRoutes(cliCtx, r, cdc)
 	auth.RegisterRoutes(cliCtx, r, cdc, "acc")
-	bank.RegisterRoutes(cliCtx, r, cdc, kb)
-	stake.RegisterRoutes(cliCtx, r, cdc, kb)
-	slashing.RegisterRoutes(cliCtx, r, cdc, kb)
+	bank.RegisterRoutes(cliCtx, r, cdc, func() (crkeys.Keybase, error) { return keys.GetKeyBase() })
```

Comment:
Might as well define this function as a variable instead of repeating it several times